src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -160,6 +152,9 @@ private static void Run(string[] args)
 
         BuildOptionsFromConfigFiles(app);
 
+        bool validArgs = ValidateArguments(args, app);
+        if (!validArgs) return;
```

Comment:
minor: would drop `validArgs` variable it doesn't add anything.
```suggestion
        if (!ValidateArguments(args, app)) return;
```